styx-scheduler-service/src/test/java/com/spotify/styx/SystemTest.java

```diff
@@ -64,23 +64,29 @@ public void shouldCatchUpWithNaturalTriggers() throws Exception {
     givenNextNaturalTrigger(HOURLY_WORKFLOW, "2016-03-14T13:00:00Z");
 
     styxStarts();
-    timePasses(1, SECONDS);
+    tickTriggerManager();
+    Thread.sleep(100);
```

Comment:
Can this be implemented without a sleep? Perhaps use awaitility to wait for the condition you had in mind.